client/src/featureform/register.py

```
@@ -4825,6 +4834,7 @@ def __init__(
         self._host = host
         self._cert_path = cert_path or os.getenv("FEATUREFORM_CERT")
         self._insecure = insecure
+        self.logger = logging.getLogger(__name__)
```

Ruff (0.8.2)

`logging` may be undefined, or defined from star imports

(F405)